lib/cdo/delete_accounts_helper.rb

```diff
@@ -125,6 +123,26 @@ def clean_and_destroy_pd_content(user_id, user_email)
         WHERE `pd_teacher_applications`.`user_id` = #{user_id}
       SQL
     )
+
+    # SQL query to anonymize Pd::FacilitatorProgramRegistration because the model no longer exists
+    ActiveRecord::Base.connection.exec_query(
+      <<-SQL
+        UPDATE `pd_facilitator_program_registrations`
+        SET `pd_facilitator_program_registrations`.`form_data` = ''
+        WHERE `pd_facilitator_program_registrations`.`user_id` = #{user_id}
+    SQL
+    )
+
```

Comment:
I am one of the people here against this coding pattern I have seen. 

1) Encapsulate this into a method
def anonimizeFacilitatorProgramRegistration()


By doing this, you can read the logic through code and also keeps the method from growing a lot (the pattern I don't like).

My practice the [rule of 30](https://www.linkedin.com/pulse/rule-30-when-your-method-class-packages-too-big-maaz-rehman)

Reply:
Also if you create the method with future mantainability, you can add a map as parameter where you pass the fields that need to be anonymized.
This would allow to add new anonymized fields in the future. But that's a stretch here, I just thought of mentioning it here